src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObj = {};
+  const styles = sourceString.replaceAll(/[\r\n\s]+/g, '').split(';');
+
+  for (let i = 0; i < styles.length; i++) {
+    const properties = styles[i].split(':');
```

Comment:
When splitting the style string by ':', there is a possibility that `properties` might not have both key and value, especially if the input string is malformed or contains extra semicolons. Consider adding a check to ensure `properties` has exactly two elements before accessing them.